src/client/getProviderUrl.ts

```diff
@@ -0,0 +1,30 @@
+const hostByChainId: Record<number, string> =

```

Comment:
Recommend using `apiUrls` from `@airswap/constants` for these.
https://github.com/airswap/airswap-protocols/blob/develop/tools/constants/index.ts#L85

Reply:
using lib and fallback now, thanks :)

Reply:
Please remove the fallback as AirSwap only supports networks found in `apiUrls`.